docs/configuring-archival.md

```diff
@@ -0,0 +1,135 @@
+---
+id: configuring-archival
+title: Configuring Archival
+---
+ 
+## What is Archival?
+
+Archival is a feature that automatically moves Workflow Event Histories from Temporal's normal persistence storage to a long-term blob store after the Workflow retention period expires.
+
+This feature enables Event Histories to be stored indefinitely while not overwhelming the persistence store.
+
+The Archival feature can be customized per Namespace, but must be enabled and configured at the Cluster level as well.
+
+:::note
+
+**Archival only supports Event Histories.**
+
+You may see some infrastructure in place to support archiving visibility records. However, the functionality is not yet supported and visibility records are currently deleted after the Workflow retention period.
+
+:::
+
+## Why use Archival?
+
+There are two main reasons you may want to keep Event Histories after the retention period has past:
+
+1. **Compliance:** For legal reasons Event Histories may need to be stored for a long period of time.
+2. **Debugging:** Old Event Histories may help with debugging.
+
+:::info
+
+**Want to turn off the Archival feature?**
+
+Set `archival.history.state` and `namespaceDefaults.archival.history.state` to `disabled`.
+
+:::
+
+## How to use Archival
+
+There are four main components to the Archival feature that you need to be aware of.
+
+1. **Archival feature configuration**: The Archival feature is controlled by the `archival` configuration in the `config/development.yaml` file.
+2. **Archive Provider**: The location where the Event Histories are archived to is called the Archive Provider. Common providers are S3, GCloud, Kafka, and the local file system.
+3. **Archiver**: The actual system component that archives and retrieves Event Histories is called an Archiver. An Archiver is directly correlated to a single Archive Provider. 
+4. **URI**: The URI is what the Namespace uses to know which Archive Provider and Archiver to use. The schema of the URI identifies which Archiver and Provider to use, and where the URI points to, identifies the actual archive location.
+
+Temporal supports several Archive Providers (and Archivers) out of the box as well as the option to create and use your own. If you want to create your own Archiver, follow [this guide](https://github.com/temporalio/temporal/blob/master/common/archiver/README.md).
+
+New Namespaces require the Archival URI to be set at the time of creation. Once set, the URI can not be changed for a Namespace. Each Namespace supports a single Archival URI, but each Namespace in a cluster can have a unique URI.
+
+When creating a new Namespace, if the URI is not specifically provided, the Namespace will use the value of `defaultNamespace.archival.history.URI` from the `config/development.yaml` file.
```

Comment:
Merge this with the previous sentence which will reduce info for user and make it accurate.